scripts/preprocess_spec.py

```diff
@@ -102,7 +102,17 @@ def strip_tags_from_operation_id(operation, _):
     operation['operationId'] = operation_id
 
 
+def add_thirdparty_resource_paths(spec):
+    with open('thirdpartypaths.json', 'r') as third_party_spec_file:
+        third_party_spec = json.loads(third_party_spec_file.read())
+    for path in third_party_spec.keys():
+        spec['paths'][path] = third_party_spec[path]
```

Comment:
Can you just add a validation here to make sure we are not overwriting an existing path?